features/product/viewing_product_in_admin_panel/viewing_product_catalog_price_history_within_product_variant.feature

```diff
@@ -0,0 +1,37 @@
+@viewing_products
+Feature: Seeing catalog price history within a variant
+    In order to be aware of historical variant prices
+    As an Administrator
+    I want to browse the catalog price history of a specific variant
+
+    Background:
+        Given the store operates on a single channel in "United States"
+        And the store has a "Wyborowa Vodka" configurable product
+        And the product "Wyborowa Vodka" has a "Wyborowa Vodka Exquisite" variant priced at "$40.00"
+        And the product "Wyborowa Vodka" has a "Wyborowa Vodka Lemon" variant priced at "$10.00"
+        And there is disabled catalog promotion "Winter sale" with priority 1 that reduces price by "50%" and applies on "Wyborowa Vodka Exquisite" variant
+        And there is disabled catalog promotion "Christmas sale" with priority 2 that reduces price by fixed "$5.00" in the "United States" channel and applies on "Wyborowa Vodka" product
+        And the "Wyborowa Vodka Exquisite" variant is now priced at "$45.00" and originally priced at "$15.00"
```

Comment:
Original price is lower than the catalog price 🤔 
Edit: I checked it and indeed it is possible to create such a "promotion" 😃 
Edit 2:
I probably found a bug:
![image](https://user-images.githubusercontent.com/40125720/214229476-2585e5f9-9ed0-4e01-9e0e-c1dcd2a12180.png)
![image](https://user-images.githubusercontent.com/40125720/214229499-bba35194-c712-4feb-b78f-7900a58e3e89.png)

Reply:
What is a bug here? 🤔

Reply:
Oh I see, original price should be crossed out but it's even not displayed. @CoderMaggie Is it a bug? 😄